crates/forge/bin/cmd/test/mod.rs

```diff
@@ -814,6 +827,49 @@ fn persist_run_failures(config: &Config, outcome: &TestOutcome) {
     }
 }
 
+/// Generate test report in JUnit XML report format.
+fn junit_xml_report(results: &BTreeMap<String, SuiteResult>, verbosity: u8) -> Report {
+    let mut junit_report = Report::new("Test run");
+    junit_report.set_timestamp(Utc::now());
+    for (suite_name, suite_result) in results {
+        let mut test_suite = TestSuite::new(suite_name);
+        test_suite.set_system_out(suite_result.summary());
+        for (test_name, test_result) in &suite_result.test_results {
+            let test_status = if test_result.status.is_failure() {
+                let mut status = TestCaseStatus::non_success(NonSuccessKind::Failure);
```

Comment:
the other kind is Error, which would be if it failed expectedly with testFail... I don't think we have to bother with this, but just noting